src/chimera/interfaces/focuser.py

```diff
@@ -26,7 +26,20 @@
 
 FocuserFeature = Enum("TEMPERATURE_COMPENSATION",
                       "ENCODER",
-                      "POSITION_FEEDBACK")
+                      "POSITION_FEEDBACK",
+                      "CONTROLLABLE_X",
+                      "CONTROLLABLE_Y",
+                      "CONTROLLABLE_U",
+                      "CONTROLLABLE_V",
+                      "CONTROLLABLE_W")
+
+FocuserAxis = Enum("X", "Y", "Z", "U", "V", "W")
+
+ControllableAxis = {FocuserFeature.CONTROLLABLE_X: FocuserAxis.X,
+                    FocuserFeature.CONTROLLABLE_Y: FocuserAxis.Y,
+                    FocuserFeature.CONTROLLABLE_U: FocuserAxis.U,
+                    FocuserFeature.CONTROLLABLE_V: FocuserAxis.V,
+                    FocuserFeature.CONTROLLABLE_W: FocuserAxis.W}
```

Comment:
No... Z is the default... A focuser without Controllable_Z is not a
focuser...


T.

On Mon, Oct 26, 2015 at 12:09 PM, William Schoenell <
notifications@github.com> wrote:

> In src/chimera/interfaces/focuser.py
> <https://github.com/astroufsc/chimera/pull/117#discussion_r43005134>:
>
> > @@ -26,7 +26,20 @@
> >
> >  FocuserFeature = Enum("TEMPERATURE_COMPENSATION",
> >                        "ENCODER",
> > -                      "POSITION_FEEDBACK")
> > +                      "POSITION_FEEDBACK",
> > +                      "CONTROLLABLE_X",
> > +                      "CONTROLLABLE_Y",
> > +                      "CONTROLLABLE_U",
> > +                      "CONTROLLABLE_V",
> > +                      "CONTROLLABLE_W")
> > +
> > +FocuserAxis = Enum("X", "Y", "Z", "U", "V", "W")
> > +
> > +ControllableAxis = {FocuserFeature.CONTROLLABLE_X: FocuserAxis.X,
> > +                    FocuserFeature.CONTROLLABLE_Y: FocuserAxis.Y,
> > +                    FocuserFeature.CONTROLLABLE_U: FocuserAxis.U,
> > +                    FocuserFeature.CONTROLLABLE_V: FocuserAxis.V,
> > +                    FocuserFeature.CONTROLLABLE_W: FocuserAxis.W}
>
> Did you forgot the "CONTROLLABLE_Z"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astroufsc/chimera/pull/117/files#r43005134>.
>